build.sbt

```diff
@@ -10,31 +10,19 @@ val scalatestVersion = "3.1.0"
 
```

Comment:
`beamVersion`, `logbackVersion`, and `uPickleVersion` can all be deleted above this line